app/models/podcast_import.rb

```diff
@@ -6,24 +6,26 @@ class PodcastImport < ApplicationRecord
 
   serialize :config, HashSerializer
 
-  attr_accessor :feed, :feed_raw_doc, :templates
+  attr_accessor :feed, :feed_raw_doc, :templates, :feed_source
 
   belongs_to :podcast, -> { with_deleted }, touch: true, optional: true, autosave: true
   has_many :episode_imports, -> { where(has_duplicate_guid: false).includes(:podcast_import) }, dependent: :destroy
 
   before_validation :set_defaults, on: :create
 
-  validates :account_id, :url, presence: true
+  validates :url, presence: true
```

Comment:
removed the validation on `account_id`, in preparation for its removal (to be done at a different time)